ui/helpers/utils/transactions.util.js

```diff
@@ -215,3 +216,12 @@ export function getTransactionTypeTitle(t, type, nativeCurrency = 'ETH') {
     }
   }
 }
+
+/**
+ * Method to check if asset standard passed is NFT
+ *
+ * @param {*} assetStandard - string
+ * @returns boolean
```

Comment:
```suggestion
 * @returns {boolean}
```

ref: https://jsdoc.app/tags-returns.html